utils/isAvailable.js

```diff
@@ -0,0 +1,7 @@
+export function isAvailable(value) {
```

Comment:
Maybe we can rename this to be more clear.

Reply:
Sure, Can you suggest a better name?